relays/ethereum/res/substrate-bridge-metadata.txt

```diff
@@ -0,0 +1,5 @@
+Last Change Date: 2020-04-28
```

Comment:
Is this date necessary? Could we not get this from the commit info?

Reply:
Commits sometimes get lost (when you move to a new repo) or we might move stuff around. I'd prefer to keep it explicitly.